examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -341,7 +365,7 @@
     "    # Compute gradient using the adjoint-state method. Note, this\n",
     "    # backpropagates the data misfit through the model.\n",
     "    grad = Function(name=\"grad\", grid=model0.grid)\n",
-    "    solver.gradient(rec=residual, u=u0, grad=grad)\n",
+    "    op.gradient(rec=residual, u=u0, grad=grad)\n",
```

Comment:
Same confusion. The solver object should be called solver. When I see op, I think Operator.

Reply:
The same thing here.